snuba/datasets/configuration/storage_builder.py

```diff
@@ -59,14 +59,14 @@ def build_storage(
 def __build_readable_storage_kwargs(config: dict[str, Any]) -> dict[str, Any]:
     return {
         STORAGE_KEY: register_storage_key(config[STORAGE]["key"]),
-        "storage_set_key": StorageSetKey(config[STORAGE][SET_KEY]),
+        "storage_set_key": register_storage_set_key(config[STORAGE][SET_KEY]),
         SCHEMA: (
             WritableTableSchema if config[KIND] == WRITABLE_STORAGE else TableSchema
         )(
             columns=ColumnSet(parse_columns(config[SCHEMA]["columns"])),
             local_table_name=config[SCHEMA]["local_table_name"],
             dist_table_name=config[SCHEMA]["dist_table_name"],
-            storage_set_key=StorageSetKey(config[STORAGE][SET_KEY]),
+            storage_set_key=register_storage_set_key(config[STORAGE][SET_KEY]),
```

Comment:
You need only register a key once, this second call to `register_storage_set_key(...)` can be replaced with `StorageSetKey(...)` since the mapping should have the key registered and available to use now

Reply:
oops, good catch